src/sonic-yang-models/tests/yang_model_tests/tests_config/feature.json

```diff
@@ -0,0 +1,66 @@
+{
+    "FEATURE_WITH_CORRECT_VALUES": {
+        "sonic-feature:sonic-feature": {
+            "sonic-feature:FEATURE": {
+                "FEATURE_LIST": [
+                    {
+                        "name": "database",
+                        "state": "always_enabled",
+                        "auto_restart": "always_enabled",
+                        "has_timer": false,
+                        "has_global_scope": true,
+                        "has_asic_scope": true
+                    },
+                    {
+                        "name": "swss",
+                        "state": "always_enabled",
+                        "auto_restart": "always_enabled",
+                        "has_timer": false,
+                        "has_global_scope": false,
+                        "has_asic_scope": true
+                    },
+                    {
+                        "name": "syncd",
+                        "state": "always_enabled",
+                        "auto_restart": "always_enabled",
+                        "has_timer": false,
+                        "has_global_scope": false,
+                        "has_asic_scope": true
+                    },
+                    {
+                        "name": "snmp",
+                        "state": "enabled",
+                        "auto_restart": "enabled",
+                        "has_timer": false,
+                        "has_global_scope": true,
+                        "has_asic_scope": false
+                    },
+                    {
+                        "name": "lldp",
+                        "state": "disabled",
+                        "auto_restart": "disabled",
+                        "has_timer": false,
+                        "has_global_scope": false,
+                        "has_asic_scope": true
```

Comment:
check the `status` field in minigraph.xml ?

Reply:
The Telemetry feature also has the same state as other feature
https://github.com/Azure/sonic-buildimage/blob/b3b6938fda9244607fb00bfd36a74bccab0c38a9/files/build_templates/init_cfg.json.j2#L39